src/io/flutter/preview/PreviewView.java

```diff
@@ -55,6 +58,9 @@
 import java.util.List;
 import java.util.Objects;
 
+import static com.intellij.icons.AllIcons.General.CollapseAll;
```

Comment:
It does not save us much, and we have space on the lines that use icons, we could just import `AllIcons`.

Reply:
I used the organize imports, but will switch to importing allicons, re-organize, and see if it's stable.